roles/icingaweb2/tasks/config_commandtransports.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: ICINGAWEB | Create monitoring module folder"
+  ansible.builtin.file:
+    path: "/etc/icingaweb2/modules/monitoring/"
+    state: directory
+    owner: "{{ icingaweb2_webserver_user }}" 
+    group: icingaweb2
```

Comment:
The group should be a variable.

Reply:
fixed! I can squash the commit if requested